python/packages/autogen-studio/autogenstudio/datamodel/db.py

```diff
@@ -47,9 +45,7 @@ class Message(SQLModel, table=True):
     session_id: Optional[int] = Field(
         default=None, sa_column=Column(Integer, ForeignKey("session.id", ondelete="CASCADE"))
     )
-    run_id: Optional[UUID] = Field(
-        default=None, sa_column=Column(SQLAlchemyUUID, ForeignKey("run.id", ondelete="CASCADE"))
-    )
+    run_id: Optional[int] = Field(default=None, sa_column=Column(Integer, ForeignKey("run.id", ondelete="CASCADE")))
```

Comment:
@ekzhu ,
I agree, it really bothered me making this switch.
But most of the usage so far is sqlite based (which does not support UUIDs natively).
I agree that at some point we should have logic that enables field types based on db support (i.e. use uuids for dbs that support it.)